azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/ConfigMojo.java

```diff
@@ -75,8 +75,8 @@ protected void doExecute() throws Exception {
         }
     }
 
-    private boolean isV1Configuration(WebAppConfiguration configuration) {
-        return configuration != null && configuration.getSchemaVersion().equals(SchemaVersion.V1.toString());
+    private boolean isV2Configuration(WebAppConfiguration configuration) {
+        return configuration == null || schemaVersion.equals(SchemaVersion.V2.toString());
```

Comment:
configuration.getSchemaVersion().equals(SchemaVersion.V2.toString()) ?

Reply:
Configuration in config mojo is generated by `V2NoValidationConfigurationParser`, which extend `V2ConfigurationParser` and will return `V2` when getSchemaVersion directly.